src/pages/build-apps/overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Build apps with Stacks
+description: Authenticate users, sign transactions and store data with the Stacks blockchain
+images:
+  large: /images/pages/build-apps.svg
+  sm: /images/pages/build-apps-sm.svg
+---
+
+## Introduction
+
+Apps built with the Stacks blockchain give users control over their digital identities, assets and data.
+
+Unlike most cloud-based apps, they are "decentralized" since they don't depend on any centralized platform, server or database to function. Rather, they use the Stacks blockchain to authenticate users and facilitate read and write transactions for them without any single point of failure or trust.
```

Comment:
```suggestion
Unlike most cloud-based apps, they are "decentralized" since they don't depend on any centralized platform, server or database to function. Rather, they use the Stacks blockchain to authenticate users and facilitate read and write requests for them without any single point of failure or trust.
```